oasis-reference/src/main/resources/com.empowerops.volition.ref_oasis/OptimizerView.fxml

```diff
@@ -0,0 +1,173 @@
+                        <Button mnemonicParsing="false" onAction="#refresh" text="Refresh"/>
+                    </HBox>
+                </VBox>
+                <padding>
+                    <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+                </padding>
+            </VBox>
+        </center>
+        <right>
+            <VBox spacing="5.0">
+                <Label text="General Actions" />
+                <GridPane hgap="5.0" vgap="5.0">
+                    <columnConstraints>
+                        <ColumnConstraints halignment="RIGHT" hgrow="SOMETIMES" maxWidth="75.0" minWidth="75.0" prefWidth="75.0" />
+                        <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+                    </columnConstraints>
+                    <rowConstraints>
+                        <RowConstraints vgrow="NEVER" />
+                        <RowConstraints vgrow="NEVER" />
+                        <RowConstraints vgrow="NEVER" />
+                        <RowConstraints vgrow="NEVER" />
+                    </rowConstraints>
+                    <Label text="Optimization" />
+                    <Label text="Configuration" GridPane.rowIndex="3" />
+                    <Button mnemonicParsing="false" onAction="#stopRun" text="Stop" GridPane.columnIndex="1" GridPane.rowIndex="1" />
+                    <Button mnemonicParsing="false" onAction="#startRun" text="Start" GridPane.columnIndex="1" />
+                    <Button disable="true" layoutX="110.0" layoutY="175.0" mnemonicParsing="false" text="Pause" GridPane.columnIndex="1" GridPane.rowIndex="2" />
+                    <Button mnemonicParsing="false" onAction="#syncAll" text="Sync All Nodes" GridPane.columnIndex="1" GridPane.rowIndex="3" />
+                </GridPane>
+                <Label layoutX="10.0" layoutY="10.0" text="Debug Actions" />
+                <GridPane hgap="5.0" layoutX="10.0" layoutY="32.0" vgap="5.0">
+                    <columnConstraints>
+                        <ColumnConstraints halignment="RIGHT" hgrow="SOMETIMES" maxWidth="75.0" minWidth="75.0" prefWidth="75.0" />
+                        <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+                    </columnConstraints>
+                    <rowConstraints>
+                        <RowConstraints vgrow="NEVER" />
+                        <RowConstraints vgrow="NEVER" />
+                    </rowConstraints>
+                    <Label layoutX="18.0" layoutY="104.0" text="Cancel All" />
+                    <Label text="Cancel-Stop" GridPane.rowIndex="1" />
+                    <Button mnemonicParsing="false" onAction="#cancelAll" text="Cancel All" GridPane.columnIndex="1" />
+                    <Button layoutX="90.0" layoutY="130.0" mnemonicParsing="false" onAction="#cancelAndStop" text="Cancel and Stop" GridPane.columnIndex="1" GridPane.rowIndex="1" />
```

Comment:
Yeah so, also theres no "force unregister" or "delete" or anything like that? Why not?  I just ran both refs, then closed the ref-simulator, and I had no way of removing the registered node from the ref optimizer.

Reply:
The old functionalities are mostly do it for every simulation since it wasn't properly use the selection model. This one add better selection it but we are also missing a 2 stage model and I generally assuming we don't have any control on that aspect and solely request on api call to registered themselves. which match what current version of oasis(this sounds a bit backward back then we don't have a lot of those idea figured out and this issues was started when we just want to refresh the overly under developed UI),  
We will be adding those back in the next issue.